CMakeLists.txt

```diff
@@ -10,12 +10,12 @@ find_package(fmt CONFIG REQUIRED)
 add_executable(main)
 target_sources(main PRIVATE src/main.cpp)
 target_link_libraries(main PRIVATE unofficial::sqlite3::sqlite3 fmt::fmt-header-only)
-set_property(TARGET main PROPERTY CXX_STANDARD 23)
+set_property(TARGET main PROPERTY CXX_STANDARD 20)
```

Comment:
i'd put back the most recent standard version. Is C++ 20 really required?

Reply:
I've fixed this issue now.
The ubuntu-latest was defaulting to `ubuntu-22.04` which didn't have a version of clang-tidy that supported c++23.
I've updated to `ubuntu24.04` and that fixed this issue.

Reply:
ok!